index.html

```diff
@@ -118,6 +123,17 @@
             "!WEBIDL#securityerror"><code>SecurityError</code></a>, and abort
             these steps.
             </li>
+            <li>Let <var>url</var> be the result of running the <a data-cite=
```

Comment:
That's a good point. I thought the URL parser was resilient to invalid URLs but it seems it has two levels of failure, "validation error" (a warning which is generally ignored) and "failure" (a parsing error).

One such fatal error is an invalid port number:

```js
navigator.share({url: 'http://localhost:65536'});
```

In Chrome, this currently fails silently, replacing the URL with the empty string and completing the successful share. Do you think this should cause navigator.share to fail with a new error? (I do.)